spec/lib/web_of_science/base_rest_retriever_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-describe WebOfScience::RestRetriever, :vcr do
-  let(:retriever) { described_class.new(params) }
+describe WebOfScience::BaseRestRetriever, :vcr do
+  let(:retriever) { described_class.new('/', params) }
 
```

Comment:
as far as I can tell, the `RestRetriever` class is not used anywhere and instead we use the `BaseRestRetriever` class ... just updating the existing spec for the unused class for the one we use